service/README.md

```diff
@@ -60,6 +60,17 @@ artifact as an "optional" dependency:
 </dependencies>
 ```
 
+In a Gradle project, one would include the `auto-service-annotations`
+artifact as a `compileOnly` dependency and `auto-service`
+as an `annotationProcessor`:
+
+```groovy
+dependencies {
+  compile             "com.google.auto.service:auto-service-annotations:${version}"
```

Comment:
Should be `compileOnly` (per text above, and because the annotation has source retention)

Reply:
oops, meant to change that, totally agree!

Reply:
Oh, fwiw, you could also use parentheses around the dependency. This is not really "idiomatic" in a Groovy DSL build, but it works and can also be copy/pasted in a Kotlin DSL file.

Reply:
works for me 👍I'll change the language of the code-block to kotlin, since that the language where the documentation is more idiomatic